cli/command/container/formatter_stats.go

```diff
@@ -13,11 +13,13 @@ const (
 	winOSType                  = "windows"
 	defaultStatsTableFormat    = "table {{.ID}}\t{{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}\t{{.MemPerc}}\t{{.NetIO}}\t{{.BlockIO}}\t{{.PIDs}}"
 	winDefaultStatsTableFormat = "table {{.ID}}\t{{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}\t{{.NetIO}}\t{{.BlockIO}}"
+	autoRangeStatsTableFormat  = "table {{.ID}}  {{.MemUsage}}  {{.AutoRange}}"
```

Comment:
Shouldn't you use `\t` between columns?
`table {{.ID}}\t{{.MemUsage}}\t{{.AutoRange}}`

Reply:
Hi @silvin-lubecki 
Concerning the lint / test / validate steps, i'm correcting those right now, should be up commited today.

For the Format, i had some trouble with values not being placed at the right spot, that's why i did it like that, but i can totally change it with \t.